shell/platform/android/BUILD.gn

```diff
@@ -490,7 +490,9 @@ action("android_jar") {
     ":pom_libflutter",
   ]
 
-  if (enable_vulkan_validation_layers) {
+  if (enable_vulkan_validation_layers ||
+      (impeller_enable_vulkan && current_cpu == "arm64" &&
+       flutter_runtime_mode == "debug")) {
     assert(impeller_enable_vulkan)
     deps += [ "//third_party/vulkan_validation_layers" ]
```

Comment:
What's the difference between this and `//third_party/vulkan-deps/vulkan-validation-layers` ?

Reply:
Oh, I see. They're the same thing https://github.com/flutter/buildroot/blob/master/build/secondary/third_party/vulkan_validation_layers/BUILD.gn.